<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextDialog</class>
 <widget class="QDialog" name="TextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Radio</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fleetIdLabel">
     <property name="text">
      <string>Fleet ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="fleetIdSpinBox">
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>349</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="deviceIdLabel">
     <property name="text">
      <string>Device ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="deviceIdSpinBox">
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>4999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="radioMessage">
     <property name="maxLength">
      <number>4096</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="broadcastCheckBox">
     <property name="toolTip">
      <string>When checked Fleet ID and Device ID are disabled.</string>
     </property>
     <property name="text">
      <string>Broadcast</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>broadcastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fleetIdSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>broadcastCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deviceIdSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
